styles/styles.css

```diff
@@ -0,0 +1,206 @@
+/* .container{
```

Comment:
General comment for entire code body: font size is a little small. Makes it a bit hard to read.